src/util/json_expr.h

```diff
@@ -18,11 +19,13 @@ class namespacet;
 
 json_objectt json(
   const exprt &,
-  const namespacet &);
+  const namespacet &,
+  const irep_idt &mode=ID_unknown);
 
 json_objectt json(
   const typet &,
-  const namespacet &);
+  const namespacet &,
+  const irep_idt &);
```

Comment:
Does it make sense to have a default value for mode to make it consistent with the version expecting an ```exprt```?